client-core/src/synchronizer/manual_synchronizer.rs

```diff
@@ -457,16 +476,26 @@ mod tests {
             heights.map(|height| self.block_results(*height)).collect()
         }
 
-        fn broadcast_transaction(&self, _transaction: &[u8]) -> Result<BroadcastTxResult> {
+        fn block_batch_verified<'a, T: Clone + Iterator<Item = &'a u64>>(
+            &self,
+            _state: lite::TrustedState,
+            _heights: T,
+        ) -> Result<(Vec<Block>, lite::TrustedState)> {
+            unreachable!()
+        }
+
+        fn broadcast_transaction(&self, _transaction: &[u8]) -> Result<BroadcastTxResponse> {
             unreachable!()
         }
 
-        fn query(&self, _path: &str, _data: &[u8]) -> Result<QueryResult> {
+        fn query(&self, _path: &str, _data: &[u8]) -> Result<AbciQuery> {
             unreachable!()
         }
     }
 
     #[test]
+    #[ignore]
+    // ignore now, mock block verification failed.
```

Comment:
instead of ignoring the test, could there be a flag (default true) to enable block verification -- in this test, it could be false?
Also, if I run a local full node myself (instead of connecting to someone else's), I may want to disable the block verification

Reply:
Done.